maptivist/src/main/webapp/filter.js

```diff
@@ -36,32 +34,36 @@ function filterByCategory(markers, markerCategory) {
 }
 
 
-function getMarkersByProximity() {
+async function getMarkersByProximity() {
     const userLongitude = document.getElementById("user-longitude");
     const userLatitude = document.getElementById("user-latitude");
-
-    fetch("/marker").then(response => response.json()).then((markers) => {
-        console.log(markers);
-        const markersToDisplay = filterByProximity(markers, userLongitude, userLatitude);
-
-    })
+    const markerRadius = document.getElementById("marker-radius");
+    const markers = await fetch("/marker").then(response => response.json());
+    
+    const markersToDisplay = filterByProximity(markers, markerRadius, userLongitude, userLatitude);
     return markersToDisplay;
 }
 
-function filterByProximity(markers, userLongitude, userLatitude) {
-    markers.sort(function (a, b) {
-        return a.distance - b.distance;
-    });
+function filterByProximity(markers, markerRadius, userLongitude, userLatitude) {
```

Comment:
What are the units on markerRadius? Miles? Kilometers? Be sure to document them! Putting the units in the variable name is another way to do this .e.g markerRadiusMiles or markerRadiusKilometers.